test/libsolidity/semanticTests/externalSource/source_remapping.sol

```diff
@@ -0,0 +1,15 @@
+==== ExternalSource: ExtSource=_external/external.sol ====
```

Comment:
I would combine `source_remapping.sol` and `source_remapping_absolute.sol` into a single test.

Also, I would use empty contracts inside. We're only testing paths here and the code inside can be minimal.